### DISCUSSION TITLE
How to make changes in a particular body and where to find  the properties in opensim creator

### DISCUSSION THREAD

**Discussion by UNNI-SANKAR** [ComputationalBiomechanicsLab/opensim-creator#903] [Q&A] [answered]
In OpenSim Creator 0.5.13 version i am facing the following issues
1. After adding a component like the pathspring, I  am not able to edit its properties (like stiffness) again which makes it difficult for re - editing. The only way for me is to use the undo option again and again to create the component from the beginning
2. Where do i find the mass and material properties and how do i change it.
3. How do i change the units ( SI to cgs ) etc.

**Comment by adamkewley** [comment, marked as answer]
1. I had difficulty reproducing your issue in 0.5.13. The following procedure worked for me:

    - Create the `PathSpring`
    - Select it (click it in the 3D viewport or `Navigator` panel)
    - Edit the `stiffness` property in the `Properties` window
    - The path spring now has a new stiffness
    - You may have other issues in your model that are preventing the property update. Check the `Log` panel for any error messages coming out of OpenSim. Otherwise, you might need to explain what you're doing in more detail for me to help.

2. The mass of each body in the model can be seen+changed by selecting a body in the model (usually, these are stored in the `bodyset` collection). Once you select a body, the `Properties` window in the model editor will show you the current mass of the body, which can then be edited.

3. OpenSim does not include unit support. The *presumed* units of each quantity may be listed under the `length_units` and `force_units` properties of your top-level model (click the top-level model node in the navigator or bottom status bar, view in the `Properties` window). The quantities you get out of the physics engine, simulator, etc. are equivalent to what you put in (e.g. if you do everything in grams, micronewtons, millimeters, then the engine will produce quantities with those units).
